cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/async/OperationFuture.java

```diff
@@ -101,4 +103,12 @@ public String progressString() {
   public OperationProgress operationProgress() {
      return _operationProgress;
   }
+
+  public void setFinishTime(long finishTime) {
+    _finishTime = finishTime;
+  }
+
```

Comment:
Missing JavaDoc -- i.e. is this an absolute time or a duration?

Reply:
this is an absolute time. the runnable associate with the future will record the finish time once it finish the task.